examples/ReceiveDemo_Advanced/output.ino

```diff
@@ -1,7 +1,7 @@
 static const char* bin2tristate(const char* bin);
 static char * dec2binWzerofill(unsigned long Dec, unsigned int bitLength);
 
-void output(unsigned long decimal, unsigned int length, unsigned int delay, unsigned int* raw, unsigned int protocol) {
+void output(unsigned long decimal, unsigned int length, unsigned int delay, volatile unsigned int* raw, unsigned int protocol) {
 
```

Comment:
How about doing an interactive rebase to squash this into the first commit introducing volatile?

(Mind you, if you don't know how to do that with git, I don't want to hold your PR hostage to that -- but it certainly woud be nicer if you were to do this :-).